docs/en/reference/sql/dql/JOIN_CLAUSE.md

```diff
@@ -18,51 +19,219 @@ JoinType ::= 'LAST'
+```
+```sql
+>CREATE TABLE t2 (id INT, col1 string,std_ts TIMESTAMP,INDEX(KEY=col1));
+SUCCEED
+>INSERT INTO t2 values(1,'a',20200520101112);
+SUCCEED
+>INSERT INTO t2 values(2,'a',20200520101113);
+SUCCEED
+>INSERT INTO t2 values(3,'b',20200520101113);
+SUCCEED
+>INSERT INTO t2 values(4,'c',20200520101114);
+SUCCEED
+>INSERT INTO t2 values(5,'b',20200520101112);
+SUCCEED
+>INSERT INTO t2 values(6,'c',20200520101113);
+SUCCEED
+>SELECT * from t2;
+ ---- ------ ----------------
+  id   col1   std_ts
+ ---- ------ ----------------
+  2    a      20200520101113
+  1    a      20200520101112
+  5    b      20200520101112
+  3    b      20200520101113
+  6    c      20200520101113
+  4    c      20200520101114
+ ---- ------ ----------------
+
+6 rows in set
+```
+The result of `SELECT` with `LAST JOIN` is shown below.
+```sql
+> SELECT * from t1 LAST JOIN t2 ON t1.col1 = t2.col1;
+ ---- ------ ---------------- ---- ------ ----------------
+  id   col1   std_ts           id   col1   std_ts
+ ---- ------ ---------------- ---- ------ ----------------
+  1    a      20200520101112   2    a      20200520101113
+  2    b      20200520101114   5    b      20200520101112
+  3    c      20200520101116   6    c      20200520101113
+ ---- ------ ---------------- ---- ------ ----------------
+
+3 rows in set
+```
+If you create t1 without index, the result of `JOIN` is the same but the order of `SELECT` result is different.
+```sql
+> SELECT * from t1 LAST JOIN t2 ON t1.col1 = t2.col1;
+ ---- ------ ---------------- ---- ------ ----------------
+  id   col1   std_ts           id   col1   std_ts
+ ---- ------ ---------------- ---- ------ ----------------
+  3    c      20200520101116   6    c      20200520101113
+  1    a      20200520101112   2    a      20200520101113
+  2    b      20200520101114   5    b      20200520101112
+ ---- ------ ---------------- ---- ------ ----------------
+
+3 rows in set
+```
+
+```{note}
+The execution of `LAST JOIN` can be optimized by index. If there is index corresponding with the `order by` and conditions in `LAST JOIN` clause, its `ts` will be used as the implicit order for unsorted `LAST JOIN`. If there is not index like this, the implicit order is the storage order. But the storage order of a table without index is unpredictable.
+If the `ts` was not given when create index, OpenMLDB uses the time when the data was inserted as `ts`. 
+```
+
+
+
 ### LAST JOIN with ORDER BY
 
-#### Example: LAST JOIN Sorting And Splicing
+#### Example2: Sorted LAST JOIN
 
-```SQL
--- desc: Simple spelling query with ORDER BY
-SELECT t1.col1 as t1_col1, t2.col1 as t2_col2 from t1 LAST JOIN t2 ORDER BY t2.std_ts ON t1.col1 = t2.col1
-```
 
-When `LAST JOIN` is configured with `Order By`, the right table is sorted by Order, and the last hit data row is spliced.
+When `LAST JOIN` is configured with `Order By`, the right table is sorted by `Order`, and the last matching data row will be joined.
```

Comment:
ORDER BY
the right table is sorted by the specified order
matching data row => matched row